index.html

```diff
@@ -97,6 +97,10 @@ <h3 class="title">DataSet</h3>
 					<li><a class="button external npm" target="_blank" href="//www.npmjs.com/package/vis-data"><span class="btn-text">npm</span></a></li>
 				</ul>
 			</section>
+
+			<section style="width: 100%">
```

Comment:
@sayzlim  Do we really need this `style="width: 100%"` here?

Reply:
@mojoaxel yes, we need width 100% here because the element itself is narrower than the section width. Without width 100%, I won’t be able to center Carbon div to match the rest of section.

The other sections don’t need 100% width because they have paragraph that fits the calculated 33% width of each section.

Reply:
On second thought, maybe we don’t need it. I saw some issue while testing though. :\

I’ll remove it.